src/components/projectRegister/StepOneForm.tsx

```
@@ -6,65 +6,34 @@ import { useState, useEffect, useRef } from "react"
-  onUpdateFormData: (data: Partial<{ targetAmount: string }>) => void
-  initialData?: {
-    category?: string
-    title?: string
-    description?: string
-    targetAmount?: string
-    startDate?: Date
-    endDate?: Date
-    deliveryDate?: Date
-    images?: ImageFile[]
-  }
+  onUpdateFormData: (data: Partial<any>) => void
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)